WTWSMS/history/titles/c_carcassonne.txt

```diff
@@ -29,21 +29,32 @@
 507.1.1={ 
 	holder=190067 #Gaisalaiks
 }
-508.1.1={ 
-	holder=63 #Theodoric 
-}
-510.1.1={
-	holder=154003 # Petrus Marcellinus Felix Liberius
-	liege = "k_italy"
-}
```

Comment:
Why did you remove these?

Reply:
My bad. Not all sources mention this, but Ostogoths did indeed capture the city. The date (508) seems a bit too early, however 😮

Reply:
Wasn't it about 518?

Reply:
OK, should be fixed now.